src/modules/Elsa.Retention/Jobs/CleanupJob.cs

```diff
@@ -0,0 +1,66 @@
+using Elsa.Common.Contracts;
+using Elsa.Retention.Extensions;
+using Elsa.Retention.Options;
+using Elsa.Workflows.Management;
+using Elsa.Workflows.Management.Entities;
+using Elsa.Workflows.Management.Enums;
+using Elsa.Workflows.Management.Filters;
+using Elsa.Workflows.Management.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Elsa.Retention.Jobs;
+
+  /// <summary>
```

Comment:
I noticed a few small issues related to extraneous white spaces (blank lines and indentation) - If you don't mind, it would be great to run some formatting to keep the files tidy.